lib/web/scripts/desktop/install-ad-ds.ps1

```diff
@@ -0,0 +1,22 @@
+$ErrorActionPreference = "Stop"
+
+$domain = 'example.com'
```

Comment:
We probably need to prompt for this - hard-coding example.com isn't a great experience.

Reply:
You're right. I've updated it so now it'll ask user for a domain name.